docs/monorepo.md

```diff
@@ -0,0 +1,23 @@
+# apollo-fetch Repository Structure
+
+`apollo-fetch` is a [lerna](https://github.com/lerna/lerna) project that contains multiple `ApolloFetch`'s including the core `apollo-fetch` package.
+
+## Commands
+
+A couple of useful commands that you will want to run in the root of the repository during your development are:
+
+* `lerna bootstrap`: installs all dependencies in packages and symlinks shared dependencies
+* `lerna run test`: tests all projects in the lerna project
+* `npm run coverage`: runs coverage for all lerna packages, depends on each module to have a `coverage:test` script.
+
+Within a project, you are welcome to include your preferred workflow.
+
+## Creating a new fetch
+
+If you are creating a new fetch and would like to included in the lerna repository, the easiest place to start is copying the `apollo-fetch` folder in `<repository root>/packages`.
```

Comment:
@stubailo I think this is intended for other contributors to PR and add in new packages if the need arises